pkg/kritis/apis/kritis/v1beta1/attestationauthority.go

```diff
@@ -33,9 +39,31 @@ type AttestationAuthority struct {
 
 // AttestationAuthoritySpec is the spec for a AttestationAuthority resource
 type AttestationAuthoritySpec struct {
-	NoteReference string   `json:"noteReference"`
-	PublicKeyList []string `json:"publicKeyList"`
-	PolicyType    string   `json:"policyType"`
+	NoteReference string      `json:"noteReference"`
+	PublicKeys    []PublicKey `json:"publicKeys"`
+	PolicyType    string      `json:"policyType"`
+}
+
+// PublicKey stores key data used to verify Attestations.
+type PublicKey struct {
+	// KeyId is the ID of this public key. This is a required field for all keys.
+	KeyId string `json:"keyId"`
```

Comment:
Apologies, I was basing the draft PR off of the wrong notes. We decided that this would be optional for PGP keys. I have made the necessary changes to implement this